staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -106,6 +103,10 @@ func createHandler(r rest.NamedCreater, scope RequestScope, typer runtime.Object
 		// TODO: replace with content type negotiation?
 		includeUninitialized := req.URL.Query().Get("includeUninitialized") == "1"
 
+		requestReceived := request.TimeStampFrom(ctx)
```

Comment:
ReceivedTimeStampFrom maybe? Makes it clearer.

Reply:
ok